scripts/enable-resource-locking.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+JSONPATH="[?(contains(type, 'Storage') || contains(type, 'KeyVault') || contains(type, 'SQL') || contains(type, 'Insights')) ].[resourceGroup]"
+
+echo "retrieve all resource groups in a subscription"
+RG_LIST=$(az resource list --query "${JSONPATH}"  -o tsv | sort -u)
+
+
+for rg in ${RG_LIST[@]}
+do
+  RG_NAME=$(sed -e 's/^"//' -e 's/"$//' <<<"$rg")
```

Comment:
As discussed on Slack - Can you check if $RG_NAME needed? Should be able to just use $rg